config/billing_rates.yml

```diff
@@ -41,7 +41,7 @@
 - { id: 159ffbe1-16ba-8978-aa38-164b7230d392, resource_type: PostgresCores,           resource_family: paradedb,            location: leaseweb-wdc02, unit_price: 0.0025448500, billed_by: duration, active_from: 2024-10-07T00:00:00Z }
 - { id: c7d24eb6-0c24-417d-8d81-f7558553f1a3, resource_type: PostgresCores,           resource_family: lantern,             location: hetzner-fsn1,   unit_price: 0.0018786380, billed_by: duration, active_from: 2024-10-10T00:00:00Z }
 - { id: df442872-a4a1-4937-9afe-7afc3f048d8b, resource_type: PostgresCores,           resource_family: lantern,             location: leaseweb-wdc02, unit_price: 0.0018786380, billed_by: duration, active_from: 2024-10-10T00:00:00Z }
-- { id: 46db9ff6-7f5a-4942-bd1a-6feccfe6c970, resource_type: PostgresStandbyCores,    resource_family: standard,            location: hetzner-fsn1,   unit_price: 0.0012524250, billed_by: duration, active_from: 2023-01-01T00:00:00Z }
```

Comment:
Should we introduce a new billing rate for this typo fix? It will change the total usage of the billing records, causing discrepancies between the invoices and the billing records.